lib/event/eventFields.js

```diff
@@ -76,6 +77,10 @@ export const EventType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Filename of an image uploaded to a 3rd party hosting service',
     },
+    location: {
+      type: LocationType,
+      description: 'The address of the event in both postal and coordinate form',
```

Comment:
Yup. I have broken it further down now so it is now:
location -> address, coordinates -> lat, long